packages/build/tests/edge_functions/snapshots/tests.js.md

```diff
@@ -420,6 +488,28 @@ Generated by [AVA](https://avajs.dev).
     ␊
     (Edge Functions bundling completed in 1ms)␊
     ␊
+    ────────────────────────────────────────────────────────────────␊
+      2. Edge Functions manifest validation                         ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    {␊
+      "bundles": [␊
+        {␊
+          "asset": "HEXADECIMAL_ID.js",␊
+          "format": "js"␊
+        }␊
+      ],␊
+      "routes": [␊
+        {␊
+          "function": "function-1",␊
+          "pattern": "^/one/?$"␊
+        }␊
+      ],␊
+      "bundler_version": "1.0.0"␊
+    }␊
```

Comment:
Should we always log the manifest? I wonder if this is useful for the user at all if it is valid.

Reply:
I don't think we should.